uuid.go

```diff
@@ -83,6 +83,10 @@ var (
 )
 
 // Equal returns true if a and b are equivalent.
+//
+// Deprecated: this function is deprecated and will be removed in a future major
+// version. The UUID type itself is directly comparable to another UUID value
+// using `==`, meaning this function superfluous.
```

Comment:
"meaning this function superfluous" does not sound quite right.

Reply:
Perhaps more of the phrase can be simplified by saying "values of type UUID are comparable using ==".